src/utils/date.tsx

```diff
@@ -1,7 +1,6 @@
 import { getDateDisplayValue } from '@terraware/web-components/utils';
-import moment from 'moment';
 export default getDateDisplayValue;
 
 export const getTodaysDateFormatted = () => {
-  return moment().format('YYYY-MM-DD');
```

Comment:
not sure what moment was returning maybe not utc

Reply:
other modals use default today's date as `getDateDisplayValue(Date.now())`
open to other suggestions but prefer one code fragment for consistency

Reply:
@constanzauanini fyi

Reply:
We should delete this helper and just use getDateDisplayValue(Date.now()) everywhere. But this is fine for now if we don't want to replace it in this PR

Reply:
I was thinking we could replace all usages of `getDateDisplayValue(Date.now())` with `getTodaysDateFormatted()` -  to be more high level. Just want to make sure you are ok with this plan of not using the `moment` based code.

Reply:
Yes, Date is js native, so I think less conversions is better